scheduler/bin/make-gke-test-cluster

```diff
@@ -13,41 +13,10 @@ PROJECT=$1
 ZONE=$2
 CLUSTERNAME=$3
 
-echo "---- Building kubernetes cluster for project=$PROJECT zone=$ZONE clustername=$CLUSTERNAME"
+bin/help-delete-temporary-clusters $ZONE
+bin/help-make-cluster $PROJECT $ZONE $CLUSTERNAME .cook_kubeconfig_1
```

Comment:
The `help-` at the beginning of these seems confusing. Can we remove it?

Reply:
I want some way to indicate that these are not supposed to be used directly. help-* prefix isn't unreasonable to do this.

Reply:
Ah, got it. The way we've done this historically is to create bash functions in a separate file, e.g. `util.sh`